packages/wonder-blocks-data/src/index.js

```diff
@@ -23,79 +13,16 @@ export type {
     ValidCacheData,
 } from "./util/types.js";
 
-/**
- * Initialize the hydration cache.
- *
- * @param {ResponseCache} source The cache content to use for initializing the
- * cache.
- * @throws {Error} If the cache is already initialized.
- */
-export const initializeCache = (source: ResponseCache): void =>
-    SsrCache.Default.initialize(source);
-
-/**
- * Fulfill all tracked data requests.
- *
- * This is for use with the `TrackData` component during server-side rendering.
- *
- * @throws {Error} If executed outside of server-side rendering.
- * @returns {Promise<void>} A promise that resolves when all tracked requests
- * have been fulfilled.
- */
-export const fulfillAllDataRequests = (): Promise<ResponseCache> => {
-    if (!Server.isServerSide()) {
-        return Promise.reject(
-            new Error("Data requests are not tracked when client-side"),
-        );
-    }
-    return RequestTracker.Default.fulfillTrackedRequests();
-};
-
-/**
- * Indicate if there are unfulfilled tracked requests.
- *
- * This is used in conjunction with `TrackData`.
- *
- * @throws {Error} If executed outside of server-side rendering.
- * @returns {boolean} `true` if there are unfulfilled tracked requests;
- * otherwise, `false`.
- */
-export const hasUnfulfilledRequests = (): boolean => {
-    if (!Server.isServerSide()) {
-        throw new Error("Data requests are not tracked when client-side");
-    }
-    return RequestTracker.Default.hasUnfulfilledRequests;
-};
-
-/**
- * Remove the request identified from the cached hydration responses.
- *
- * @param {string} id The request ID of the response to remove from the cache.
- */
-export const removeFromCache = (id: string): boolean =>
-    SsrCache.Default.remove(id);
-
-/**
- * Remove all cached hydration responses that match the given predicate.
- *
- * @param {(id: string) => boolean} [predicate] The predicate to match against
- * the cached hydration responses. If no predicate is provided, all cached
- * hydration responses will be removed.
- */
-export const removeAllFromCache = (
-    predicate?: (
-        key: string,
-        cacheEntry: ?$ReadOnly<CachedResponse<ValidCacheData>>,
-    ) => boolean,
-): void => SsrCache.Default.removeAll(predicate);
```

Comment:
No more untested functionality hidden in index.js. These now live in `xxx- api.js` files.